docs/quickstart/actions-and-hooks.md

```diff
@@ -32,8 +32,14 @@ _Hooks_ can be either a [Lua]({% link howto/hooks/lua.md %}) script that lakeFS
                         --source lakefs://quickstart/main
     ```
 
-2. Open up your favorite text editor (or emacs), and paste the following YAML: 
-
+2. Open up a text editor 
+   ```bash
+   docker exec lakefs \
+   vi /tmp/check_commit_metadata.yml
```

Comment:
Aside from using "vi", this probably won't work for an interactive program which needs a terminal.
```suggestion
   docker exec -it lakefs \
   vi /tmp/check_commit_metadata.yml
```